example/src/app/simple-with-styles/simple-with-styles.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-simple-with-styles',
+  templateUrl: './simple-with-styles.component.html',
+  // we have to setup styles this way, since simple styles/styleUrs properties will be removed (jest does not unit test styles)
+  ['styles']: [`
```

Comment:
This is a workaround to avoid getting stripped the styles by the astTransformer.
If there are no styles to apply, Angular will not render `_ngcontent`, which we want to test here.

But a comment may be nice to explain this and not promote this syntax.

Reply:
> But a comment may be nice to explain this and not promote this syntax.

 I left comment there. Is it engough?

Reply:
Done

Reply:
For me it's good. Any developer seeing this line will understand the reason behind this.